meetings/2024-10-09.md

```diff
@@ -30,15 +39,111 @@
+* ...: demo!
+* ...: we have web version on left and native linux on the right. there are three stores, one is private which user can only share across devices. there is also the shared space, where one can interact with other users. There is also a public space, like a website.
+* ...: I will create a new document, we can select from different primary classes. As you see it synchronized. I can change the title, it also updated in the other app. I can do a SPARQL update.
+* ...: you can see `did:ng:o:.....` the last part has a public key. 
+* ...: let's do another update, now in the other app we see the whole history of the document, creation, signature, adding first triple, then adding another triple. I will add another one on the left and we can see another commit on the right. 
+* ...: Now I will disable connection in the web app, I can still edit the document. And in the other ap I will make a different update. No sync yet since the webapp is offline. Now we go backonline and we see on the right a fork. As soon as I make another commit we can see the automatic merge. So it has the automerge without conflicts, it also work on data deletion.
+* Rui: what does the `o` mean in the DID identifier?
+* Niko: We are registering the `did:ng` method, the rest is specific to the next graph and `o` means a document. You can find it in the DID section of NextGraph docs
+* Rahul: Are you aware of Georges m-ld? How does it compare. Also what transport are you using? Are you aware of CRDTs algorithms developed by braid?
+* Niko: Good remarks, I'm familiar with m-ld. I still need to talk with George. m-ld is not end-to-end encrypted but we need to discuss details. We use websockets for now on every clients, also between brokers we use websockets for practical reasons. As for braid, I met them a week ago, I will join them again in the week. A lot of common grounds and interest to work together. 
+* Tim: Does it use UDP or TCP
+* Niko: it uses WebSockets so TCP. For now it's good.
+* Tim: This is very exciting, it has a lot of things we want in Solid. We want local-first. Where are you based and who do you work for, why did you start NextCloud.
+* Niko: I'm independent, sponsored by NLnet using EU grants last two years. I applied for another grant this year. I really like your vision about the giant global graph, very inspired by that. I worked with property graph first, four years I embraced RDF. Later added CRDTs and end-to-end encryption. Trying to bridge a gap with solid.
+* Hadrian: Is new method neccessary? There is `did:key`, you may have problem with key rotation.
+* Niko: yes key rotation is very important, it is already integrated into protocol and the design. The public key is never going to change, but it is not used for signing. There is a chain, this can be revoked and updated.
+* Hadrian: a discussion for another time
+* Niko: please feel invited to nextgraph forum
```

Comment:
```suggestion
* Niko: yes, key rotation is very important. It is already integrated into the protocol and the design. The public key is never going to change, but it is not used for signing. There is a chain; this can be revoked and updated.
* Hadrian: A discussion for another time.
* Niko: Please feel invited to NextGraph forum.
```